mcs/class/corlib/corefx/MonoLinqHelper.cs

```diff
@@ -3,30 +3,11 @@
 
 namespace System.IO
 {
-	internal static class MonoLinqHelper {
+	internal static class MonoLinqHelper
```

Comment:
Why do you need this wrapper?

Reply:
Several files expect a ToArray extension to `IEnumerable<T>` (e.g. at https://github.com/dotnet/corefx/blob/master/src/System.IO.FileSystem/src/System/IO/Directory.cs#L146) and that comes from LINQ (https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.toarray?view=netframework-4.7.2)